assembly/Hash.ts

```diff
@@ -64,8 +64,8 @@ export class Hash {
     /**
     * @description Instantiates Hash from u8[] SCALE encoded bytes
     */
-    static fromU8a (input: u8[]): Hash {
-        return new Hash(input);
+    static fromU8a (input: u8[], index: i32 = 0): Hash {
```

Comment:
maybe we should add assert for the length of the `input`? input.length - index >= 0?